core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineStrategy.java

```diff
@@ -41,6 +44,10 @@ public interface RdbEngineStrategy {
 
   String getDataTypeForKey(DataType dataType);
 
+  default String getDataTypeForSecondaryIndex(DataType dataType) {
```

Comment:
It seems like we should add the `@Nullable` annotation:
```suggestion
  @Nullable
  default String getDataTypeForSecondaryIndex(DataType dataType) {
```

Could you please add it to `String getDataTypeForKey(DataType dataType);` as well. Also, please add it to the implementation methods?

Reply:
Thank you.
f0ce19bac0ac0e1b1df8d21809a4e35fbf90d239